chatbridge/impl/discord/config.py

```diff
@@ -13,5 +13,6 @@ class DiscordConfig(ClientConfig):
 	command_prefix: str = '!!'
 	client_to_query_stats: str = 'MyClient1'
 	client_to_query_online: str = 'MyClient2'
-	embed_icon_url = 'https://cdn.discordapp.com/emojis/566212479487836160.png'
+	embed_color: int = 3447003
```

Comment:
way less readability

Reply:
ik its kinda strange, but its a way to have embed color configurable